index.js

```diff
@@ -743,8 +743,8 @@ Peer.prototype.getStats = function (cb) {
       cb(null, reports)
     }, function (err) { cb(err) })
 
-  // Unknown browser, skip getStats() since it's anyone's guess which style of
-  // getStats() they implement.
+    // Unknown browser, skip getStats() since it's anyone's guess which style of
+    // getStats() they implement.
```

Comment:
Are these indents intentional?

Reply:
Nope. I assumed it was my updated version of `standard`, but that's not the case.